systemtest/src/test/java/io/strimzi/systemtest/CustomResourceStatusST.java

```diff
@@ -56,6 +59,40 @@ void testKafkaStatus() throws Exception {
         });
     }
 
+    @Test
+    void testKafkaUserStatus() {
+        String userName = "status-user-test";
+        testClassResources().tlsUser(CLUSTER_NAME, userName).done();
+        StUtils.waitForSecretReady(userName);
+        LOGGER.info("Checking status of deployed kafka user");
+        Condition kafkaCondition = testClassResources().kafkaUser().inNamespace(NAMESPACE).withName(userName).get().getStatus().getConditions().get(0);
+        LOGGER.info("Kafka User Status: {}", kafkaCondition.getStatus());
+        LOGGER.info("Kafka User Type: {}", kafkaCondition.getType());
+        LOGGER.info("Kafka User Message: {}", kafkaCondition.getMessage());
```

Comment:
When the status is `Ready`, the `message` (and `reason`) is not set so it is unnecessary here.